extras/data/E155/metadata.yaml

```
@@ -0,0 +1,5 @@
+E155:
+  description: Measurements with Polarised beams
+  nnpdf_id: E155
+  y_label: A1
+  y_label_tex: $g1/f1$
```

```suggestion
  y_label: g1
  y_label_tex: $g1$
```
if you use this to generate both $g_1$ and $F_1$ put them one at a time and no `/` as this looks like a division